qendpoint-store/src/main/java/com/the_qa_company/qendpoint/store/HDTConverter.java

```diff
@@ -29,7 +24,7 @@ public class HDTConverter {
 	public static final String HDT_URI = "http://hdt.org/";
 	private final EndpointStore endpoint;
 	private final HDT hdt;
-	private final ValueFactory valueFactory = new MemValueFactory();
+	private final ValueFactory valueFactory = SimpleValueFactory.getInstance();
```

Comment:
Performance improvement since we don't need the deduplication aspects of the MemValueFactory.